### PR TITLE
added 3 question to JS dataset and  updated contributors.md

### PR DESCRIPTION
**Pull request by pantbibek21**
## What type of PR is this? (check all applicable)

- [✅] 🚀 Added Name
- [ ] ✨ Feature
- [✅] 🌟 stared the repo
- [ ] 🐛 Grammatical Error
- [ ] 📝 Documentation Update
- [✅] 🚩 Other

## Description
I have updated the JS question dataset with three questions and also added my details in the contributors.md file as you guided. You may delete my previous PR#82 where I forget to add my details to the contributors.md file.

## Add Link of GitHub Profile

https://github.com/pantbibek21

### CODE REVIEW

File: Languages/JS/questions.json [hunk 1]
```diff
@@ -21,30 +21,30 @@
...
-       "details": "It is a form of transport consisting of a series of connected vehicles that generally run along a railroad track to transport passengers or cargo. The word train comes from the Old French trahiner, derived from the Latin trahere meaning "to pull, to draw". ",
-       "questionType": "bool",
-       "trueOrFalse": true,
-       "shortAns": "**NOTHING GOES HERE FOR BOOLEAN QUESTIONS"="",
-       "tag": "Transportation"
+       "question": "What is NaN property in JavaScript?",
+       "details": "NaN property represents the \"Not-a-Number\" value. It indicates that the specified value is not a legal number. Example. \"a\" * 5 will result in NaN.",
+       "questionType": "multi",
+       "trueOrFalse": false,
+       "shortAns": "It is a property of JavaScript that indicates that the specified value is not a legal number.",
```
Comment (AndroidLion48) [inline Languages/JS/questions.json:46]: Your short Answer is a little to obvious.

Please reword your shorAns in order to not display main context of your question.
Example:
--What is a Cat?
--It is an animal with fur and four legs.

Wrong:
--It is a feline of the domestic variety.

**Review by AndroidLion48 — CHANGES_REQUESTED** [review]
Your short answer is to obvious.

File: Languages/JS/questions.json (hunk 1 above)
Comment (AndroidLion48) [reply Languages/JS/questions.json:46]: Just that line is fine.

Comment (Tc2r1) [reply Languages/JS/questions.json:46]: @pantbibek21  you alive?

Comment (pantbibek21) [reply Languages/JS/questions.json:46]: @AndroidLion48, sorry sir for the delay. Right into it,now.

**Comment by pantbibek21** [comment]
@AndroidLion48 sir,  I have rephrased the shortAns to be more precise.

**Review by Tc2r1 — APPROVED** [review]

**Pull request merged**